aptos-move/framework/aptos-framework/sources/dora_committee.move

```diff
@@ -0,0 +1,1042 @@
+/// Design:
+/// CommitteeInfoStore: store all the committee information, supported operation: add, remove
+/// CommitteeInfo: store all the dora node information, supported operation: add, remove
+/// DoraNodeInfo: store the dora node information, supported operation: update
+/// NodeData: store the dora node information with operator's address
+/// requirements:
+/// 1. two committee has different id
+/// 2. two indentical node should not be in the same committee
+/// 3. the operator
+module aptos_framework::dora_committee {
+    use std::error;
+    use std::signer;
+    use std::vector;
+    use aptos_std::capability;
+    use aptos_std::simple_map;
+    use aptos_std::simple_map::SimpleMap;
+    use aptos_framework::account;
+    use aptos_framework::account::new_event_handle;
```

Comment:
We can condense these four lines of code into two lines as shown below. FYI, in case you were not aware, it helps reduce the number of lines in the code
```
use aptos_std::simple_map::{Self, SimpleMap};
use aptos_framework::account::{Self, new_event_handle};
```

Reply:
Is there a tool in move to do it automatically?

Reply:
@nizam-supraoracles Thanks, addressed in new commit.

Reply:
I'm using the [Move Language plugin](https://plugins.jetbrains.com/plugin/14721-move-language) in the RutRover/CLion IDE, which partially formats the Move code.